<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="EPINPUNW03CD" failures="1" tests="1" name="TestRunnerFiles.TestRunner" time="9.080" errors="0" timestamp="2023-04-05T00:58:38 IST" skipped="0">
  <testcase classname="TestRunnerFiles.TestRunner" name="runScenario" time="9.080">
    <failure message="The following asserts failed:
	Resorce not deleted successfully expected [404] but found [200]" type="java.lang.AssertionError">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
Resorce not deleted successfully expected [404] but found [200]
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:46)
at org.testng.asserts.SoftAssert.assertAll(SoftAssert.java:30)
at pages.HomeTaskRestFunctionalTest.verifyResourceIsDeleted(HomeTaskRestFunctionalTest.java:100)
at StepDefinitions.APITestingTask.verify_that_resource_has_been_deleted_succesfully(APITestingTask.java:57)
at âœ½.Verify that resource has been deleted succesfully(file:///C:/Users/Durgesh_Honrao/Desktop/Assignments/Epam/epam_onboarding_625745/src/test/java/feature/APITestingTask.feature:11)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <system-out/>
</testsuite> <!-- TestRunnerFiles.TestRunner -->
